kernel/src/color.rs

```diff
@@ -0,0 +1,54 @@
+use core::convert::TryFrom;
```

Comment:
(About top level file). Why is this in a new module of its own? It could be in a submodule of terminal, perhaps?

Reply:
Perhaps. I was thinking these colors are more generic than just for terminal and vga. I believe changing this would be effected by how we change the use of colors in terminals. (Below comment)

Reply:
What else would we  use the colors for?

Reply:
So, we discussed this a bit IRL and came to the conclusion that we will expand the color module sometime (probably moving it into another crate or something)